koku/providers/aws/aws_provider.py

```diff
@@ -124,14 +124,16 @@ def _check_org_access(access_key_id, secret_access_key, session_token):
     return access_ok
 
 
-def _check_cost_report_access(access_key_id, secret_access_key, session_token):
+def _check_cost_report_access(access_key_id, secret_access_key, session_token,
+                              region='us-east-1'):
```

Comment:
Do we know if it's possible to have the cur APIs in any other region? This is a month-old memory now, and I did not test all of the regions, but the few I did try, it looked like us-east-1 was the only one that worked with the cur client.

Reply:
us-east-1 is currently the only endpoint for CUR services. It may change later (or get dropped as a required parameter). So, making this a parameter is just hedging a little on the possibility that AWS might make this a region-specific thing.

Reply:
I believe that the IAM endpoint is only accessible in us-east maybe thats what you were remembering?  I'm looking at the boto3 and AWS docs for `cur` and I don't see anything about requiring to specify the region.  Maybe this is just undocumented?  One interesting point is that we can get the region of the S3 bucket in the response of `describe_report_definitions`.

Reply:
The CUR service stuff is very new (~3-6 months old), so it's definitely under-documented. There's only 3 CUR APIs at this time - `delete-report-definition`, `describe-report-definitions`, `put-report-definition`. 

The endpoints list (https://docs.aws.amazon.com/general/latest/gr/rande.html) doesn't even show the CUR service endpoint. But the other cost management endpoints are all exclusively in us-east-1 except the pricing API which has a new deployment into ap-south-1 (that wasn't there back in Feb/Mar).